src/pom.xml

```diff
@@ -687,12 +706,35 @@
                 <artifactId>gwc-aws-s3</artifactId>
                 <version>${geowebcache.version}</version>
             </dependency>
+            <!--
+            <dependency>
+                <groupId>org.geotools</groupId>
+                <artifactId>gt-process-spatialstatistics</artifactId>
+                <version>${gt.spatialstatistics.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.geoserver</groupId>
+                <artifactId>gs-wps-spatialstatistics</artifactId>
+                <version>${gs.spatialstatistics.version}</version>
+            </dependency>
+            -->
```

Comment:
Not much of a point adding commented out dependencies. And spatialstatistics doesn't have a public releasethat supports recent versions of geoserver right now, so Stratus is not going to include it.

Reply:
Cool.  I think one of the very first things I did was remove this because it had the funky, custom-built boundless dependencies and I wasn't sure how to handle it, then it fell off my radar.